src/main/java/QueryServlet.java

```diff
@@ -15,39 +16,94 @@
 @WebServlet("/query")
 public class QueryServlet extends HttpServlet {
 
-  // Hardcoded initial choices for which lines of the data table are accessed
+  // Hardcoded initial choices for which lines of which data table are accessed
   Map<String, Map<String, String>> queryToDataRow =
       ImmutableMap.of(
           "live",
-          ImmutableMap.of("under-18", "0019E", "over-18", "0021E", "all-ages", "0001E"),
+          ImmutableMap.of(
```

Comment:
I know the linting recommended all these line breaks, but it doesn't seem to be recommended in the Java style guide (or could you point me to where it is?) It seems sort of inefficient to me as a formatting strategy. Happy to hear arguments otherwise however

Reply:
I agree this seems kinda inefficient, but I do think it looks cleaner, and as we add more query dropdowns these maps will only get longer and we'd have to break the maps into multiple lines anyway. I don't see this mentioned in the written style guide however the linter I added is straight from google so I'm guessing anything the linter suggests aligns with the style guide

Reply:
OK, makes sense!